src/map/status.c

```diff
@@ -6327,10 +6327,17 @@ static void status_set_viewdata(struct block_list *bl, int class_)
 	case BL_MOB:
 	{
 		struct mob_data *md = BL_UCAST(BL_MOB, bl);
-		if (vd != NULL)
+		if (vd) {
```

Comment:
change that back to vd != NULL.

vd is not a boolean-typed variable...

Reply:
Uploaded changes